### ISSUE TITLE
English Readme

### ISSUE THREAD

**Issue by alirezatheh** [deku772/ASUS-A450J-X450JF-N550jv--EFI#3]
can you please create an English version of your readme, hdmi and sip?

**Comment by deku772** [comment]
In fact, I only provide the most basic solution, and you can also find corresponding solutions on foreign websites.
Secondly, if you dabble a little, you will find that following my installation method step by step, there should be no problem. After all, I have personally tested it many times.

**Comment by alirezatheh** [comment]
Yes, but it is hard to understand when I translate it from Chinese.
